core/modules/loader/CentralWorker.cc

```diff
@@ -382,7 +382,7 @@ void CentralWorker::_shift(Direction direction, int keysToShift) {
 
             // TODO This is very similar to code in TcpBaseConnection::_handleShiftToRight and they should be merged.
             int sz = protoKeyList->keypair_size();
-            std::vector<CentralWorker::StringKeyPair> keyList;
+            std::vector<CentralWorker::CompKeyPair> keyList;
```

Comment:
I would highly recommend to get rid of that `std::` in the CC-files. This will eliminate a lot of clutter from the code. Just put somewhere on top of the file:
```
using namespace std;
```